packages/paste-design-tokens/tokens/global/sizing.yml

```diff
@@ -7,157 +7,189 @@ imports:
     value: "{!size-square-40}"
-    comment: Generic square sizing token scale for UI components.
+    comment: Generic sizing token for layouts and containers
   size-square-50:
     value: "{!size-square-50}"
-    comment: Generic square sizing token scale for UI components.
+    comment: Generic sizing token for layouts and containers
   size-square-60:
     value: "{!size-square-60}"
-    comment: Generic square sizing token scale for UI components.
+    comment: Generic sizing token for layouts and containers
   size-square-70:
     value: "{!size-square-70}"
-    comment: Generic square sizing token scale for UI components.
+    comment: Generic sizing token for layouts and containers
   size-square-80:
     value: "{!size-square-80}"
-    comment: Generic square sizing token scale for UI components.
+    comment: Generic sizing token for layouts and containers
   size-square-90:
     value: "{!size-square-90}"
-    comment: Generic square sizing token scale for UI components.
+    comment: Generic sizing token for layouts and containers
   size-square-100:
     value: "{!size-square-100}"
-    comment: Generic square sizing token scale for UI components.
+    comment: Generic sizing token for layouts and containers
   size-square-110:
     value: "{!size-square-110}"
-    comment: Generic square sizing token scale for UI components.
+    comment: Generic sizing token for layouts and containers
   size-square-120:
     value: "{!size-square-120}"
-    comment: Generic square sizing token scale for UI components.
+    comment: Generic sizing token for layouts and containers
   size-square-130:
     value: "{!size-square-130}"
-    comment: Generic square sizing token scale for UI components.
+    comment: Generic sizing token for layouts and containers
   size-square-140:
     value: "{!size-square-140}"
-    comment: Generic square sizing token scale for UI components.
+    comment: Generic sizing token for layouts and containers
   size-square-150:
     value: "{!size-square-150}"
-    comment: Generic square sizing token scale for UI components.
+    comment: Generic sizing token for layouts and containers
   size-square-160:
     value: "{!size-square-160}"
-    comment: Generic square sizing token scale for UI components.
+    comment: Generic sizing token for layouts and containers
   size-square-170:
     value: "{!size-square-170}"
-    comment: Generic square sizing token scale for UI components.
+    comment: Generic sizing token for layouts and containers
   size-square-180:
     value: "{!size-square-180}"
-    comment: Generic square sizing token scale for UI components.
+    comment: Generic sizing token for layouts and containers
   size-square-190:
     value: "{!size-square-190}"
-    comment: Generic square sizing token scale for UI components.
+    comment: Generic sizing token for layouts and containers
   size-square-200:
     value: "{!size-square-200}"
-    comment: Generic square sizing token scale for UI components.
```

Comment:
suggestion(blocking): these comments should not be changes. The description form Jira ticket should only be applied to new tokens